### PR TITLE
ci: Add next images build

### PR DESCRIPTION
**Pull request by mkuznyetsov**
Signed-off-by: Mykhailo Kuznietsov <mkuznets@redhat.com>

### What does this PR do?
Use next images instead of nightly for 'nightly' channel

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [Custom resource definition file is up to date](https://github.com/eclipse-che/che-operator/blob/main/README.md#updating-custom-resource-definition-file)
- [ ] [Nightly OLM bundle is up to date](https://github.com/eclipse-che/che-operator/blob/main/README.md#update-nightly-olm-bundle)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Comment by tolusha** [comment]
We still have a lot of references all over the bash scripts and readme files.

**Comment by flacatus** [comment]
@mkuznyetsov Instead of creating a new workflow... Will not be more easy to add the new tag here: https://github.com/eclipse-che/che-operator/blob/main/.github/workflows/build-nightly-images.yaml#L45 ?

**Comment by mkuznyetsov** [comment]
@flacatus I wanted to add new workflow, to have next displayed in a new tab (the workflow will also have a different name). Then

**Comment by AndrienkoAleksandr** [comment]
Should we swich olm channel from nightly to next ?

> We still have a lot of references to nightly all over the bash scripts and readme files.

Some part of this references related to the nightly channel. And release scripts too.

**Comment by tolusha** [comment]
@AndrienkoAleksandr 
We can but it will require changes in chectl as well.
Let's change image from `nightly` to `next` first and consider changing channel.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ che-operator Development Guide: https://github.com/eclipse-che/che-operator/#dev
 
 
 ### PR Checklist
-
+
```
Comment (nickboldt) [inline .github/PULL_REQUEST_TEMPLATE.md:26]: weird extra space here

File: .ci/oci-disconnected.sh [hunk 2]
```diff
@@ -27,7 +27,7 @@ export SLACK_TOKEN="${SLACK_TOKEN-"UNDEFINED"}"
 export WORKSPACE="${WORKSPACE-"UNDEFINED"}"
 export REG_CREDS=${XDG_RUNTIME_DIR}/containers/auth.json
 export ORGANIZATION="eclipse"
-export TAG_NIGHTLY="nightly"
+export TAG_NIGHTLY="next"
```
Comment (mmorhun) [inline .ci/oci-disconnected.sh:30]: Shouldn't we rename `TAG_NIGHTLY` into `TAG_NEXT` as well?

**Review by flacatus — APPROVED** [review]

**Pull request merged**